### DISCUSSION TITLE
ESLINT 9 SUPPORT??

### DISCUSSION THREAD

**Discussion by Pnlvfx** [vercel/next.js#69496] [App Router] [answered]
Do nextjs plan to support eslint 9?
I've seen that eslint-plugin-react-hooks canary and eslint-plugin-react now supports eslint 9, 
Eslint also add some compatibility for libraries to still support old apis during the migration.

I love the new eslint config and I'm looking to any of your commits to hope that it will be implemented soon, that's why I'm asking, if there is no plan for now, I will probably try to find a workaround disabling the next.js config even if I prefer to avoid it.

**Comment by Juneezee** [comment]
Unfortunately, Next.js doesn't support ESLint 9 yet. There is an issue tracking this, see https://github.com/vercel/next.js/issues/64409

**Comment by leerob** [comment, marked as answer]
ESLint 9 is supported as of the Next.js 15 RC 2. v15 stable release coming soon 🙏 

https://x.com/nextjs/status/1846276572812124565
